Memories/Memories/MCs/Helper.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Helper.swift
+//  Test
+//
+//  Created by Alexander Supe on 13.03.20.
+//  Copyright © 2020 Alexander Supe. All rights reserved.
+//
+
+import UIKit
+
+struct Helper {
```

Comment:
Use a better name for what this structure is doing.

Helper is too generic.

You're saving/loading lots of image data to the UserDefaults, which is not recommended.

For the sprint challenge this is sufficient for persistence, but this does not scale well and slows down access to your "quick access" user default database. Image data should be either saved to disk as images, or within a core data entity.